examples/hmm.py

```diff
@@ -414,14 +414,14 @@ def main(args):
         torch.set_default_tensor_type('torch.cuda.FloatTensor')
 
     logging.info('Loading data')
-    data = poly.load_data()
+    data = poly.load_data(poly.JSB_CHORALES)
 
     logging.info('-' * 40)
     model = models[args.model]
     logging.info('Training {} on {} sequences'.format(
         model.__name__, len(data['train']['sequences'])))
-    sequences = torch.tensor(data['train']['sequences'], dtype=torch.float32)
-    lengths = torch.tensor(data['train']['sequence_lengths'], dtype=torch.long)
+    sequences = data['train']['sequences']
```

Comment:
where in the preprocessing did this become an array and not a list of sequences?

Reply:
This is happening inside `load_data` when you call `load_data(poly.JSB_CHORALES)`.